### PR TITLE
Playitgg integration

### PR DESCRIPTION
**Pull request by seliayeu**
add playitgg integration

### CODE REVIEW

File: core/src/handlers/global_settings.rs [hunk 1]
```diff
@@ -111,11 +111,35 @@ pub async fn change_domain(
...
+    axum::extract::State(state): axum::extract::State<AppState>,
+    AuthBearer(token): AuthBearer,
+    Json(playit_enabled): Json<bool>,
+) -> Result<(), Error> {
+    let requester = state.users_manager.read().await.try_auth_or_err(&token)?;
+
+    if !requester.is_owner {
+        return Err(Error {
+            kind: ErrorKind::PermissionDenied,
+            source: eyre!("Not authorized to change core safe mode"),
```
Comment (CheatCod) [inline core/src/handlers/global_settings.rs:124]: Wrong error mesage

File: core/src/playitgg/errors.rs [hunk 2]
```diff
@@ -0,0 +1,72 @@
+/* 
+Copyright 2022 Developed Methods LLC
+
+Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+
+1. Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
+
+2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
```
Comment (CheatCod) [inline core/src/playitgg/errors.rs:8]: May need to reproduce the copyright notice in our repo

File: core/src/playitgg/helper.rs [hunk 3]
```diff
@@ -0,0 +1,396 @@
...
+use playit_agent_core::network::address_lookup::{AddressLookup, AddressValue};
+use playit_agent_core::network::lan_address::LanAddress;
+use playit_agent_core::network::tcp_clients::TcpClients;
+use playit_agent_core::network::udp_clients::UdpClients;
+use playit_agent_core::tunnel::setup::SetupError;
+use playit_agent_core::tunnel::simple_tunnel::SimpleTunnel;
+use playit_agent_core::tunnel::udp_tunnel::UdpTunnelRx;
+use playit_agent_core::utils::now_milli;
+
+use crate::event_broadcaster::{self, EventBroadcaster};
```
Comment (CheatCod) [inline core/src/playitgg/helper.rs:19]: unused import `self`, clippy lint

**Review by CheatCod — CHANGES_REQUESTED** [review]
There are a bunch of clippy lints related to your change, please follow them

File: core/src/playitgg/errors.rs (hunk 2 above)
Comment (seliayeu) [reply core/src/playitgg/errors.rs:8]: I think it should be fine since we only use source code and it's credited anywhere it's used.

**Pull request merged**